Documentation/using-dex.md

```diff
@@ -137,18 +137,16 @@ Matcher symbols will only function if all of the following conditions are met:
 
 1. application is confidential (i.e. not public)
 
-2. URL scheme is `http` or `https`
```

Comment:
Do we really need to limit schema? I don't see any technical issue to support more schemas, in theory it should allow users to specify url as `my_app://path` to send code into desktop application.

Reply:
https://tools.ietf.org/html/rfc3986#section-4.3 is silent on this restriction, so no objection to remove.